coil-base/src/main/java/coil/collection/LinkedMultimap.kt

```diff
@@ -105,12 +105,12 @@ internal class LinkedMultimap<K, V>(sorted: Boolean = false) {
         entry.next.prev = entry.prev
     }
 
-    private class LinkedEntry<K, V>(@JvmField val key: K?) {
```

Comment:
Removing these since if we add them here I think it's a slippery slope to add them elsewhere internally.